api-test/README.md

```
@@ -133,15 +133,23 @@ To execute the tests using Eclipse IDE, use the following steps:
      - **VM Arguments**:
        ```
        -Dmodules=partner -Denv.user=api-internal.<env_name> -Denv.endpoint=<base_env> -Denv.testLevel=smokeAndRegression```
+	   (example : -Dmodules=Partner -Denv.user=api-internal.test-env -Denv.endpoint=https://api-internal.test-env.mosip.net -Denv.testLevel=smokeAndRegression)
+	   
+## 5. **Update the property file**
+      
+    1. Navigate to the pms.properties file located at: partner-management-services-repository\api-test\src\main\resources\config\pms.properties
```

markdownlint-cli2 (0.20.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)